ntbs-build/openshift-pipelines/config-maps/uat-config.yml

```diff
@@ -4,5 +4,10 @@ metadata:
   name: uat
   namespace: ntbs-build
 data:
-  deployment.file: phe-uat.yml
```

Comment:
This was previously wrong in the checked in version, but right in the actually deployed version.